package.json

```diff
@@ -41,6 +41,8 @@
     "body-parser": "^1.15.0",
     "commander": "^2.9.0",
     "express": "^4.0.0",
+    "fastify": "^3.24.0",
```

Comment:
I do not remember why "express" in the prod deps, but "koa" and "hapi" are in dev deps.
Could you please move "fastify" to dev deps too?

Reply:
it's ok for you now ? 😃

Reply:
@koresar 🙏